openstef/model/regressors/linear_quantile.py

```diff
@@ -177,6 +184,7 @@ def fit(self, x: pd.DataFrame, y: pd.Series, **kwargs) -> RegressorMixin:
             y = pd.Series(np.asarray(y), name="load")
 
         x = self._remove_ignored_features(x)
+        x = self.feature_clipper_.fit_transform(x)
```

Comment:
Why would we `fit_transform` here rather than just `fit`. I know transform doesn't change anything here, but then why would we add it?

Reply:
Good one. Fixed, transforming during training indeed doesn't do anything.